openfl/federated/task/runner_keras.py

```diff
@@ -60,7 +60,7 @@ def rebuild_model(self, round_num, input_tensor_dict, validation=False):
         """
         if self.opt_treatment == "RESET":
             self.reset_opt_vars()
-            self.set_tensor_dict(input_tensor_dict, with_opt_vars=False)
+            self.set_tensor_dict(input_tensor_dict, with_opt_vars=True)
```

Comment:
Intuitively, if the opt treatment is "RESET", shouldn't we have `with_opt_vars=False` ?

Reply:
This is done as when optmizers weights are reset. Leraning rate is zero and model accuracy remiains arnd ~0.125.

Reply:
Once `reset_opt_vars()` is fixed, let's set this back to `with_opt_vars=False` since the intention is to not rebuild the model with the global optimizer (and "reset" it back to the local optimizer)

Reply:
done